tests/smoke_tests/smoke_tests_utils.py

```diff
@@ -622,7 +622,7 @@ def get_gcp_region_for_quota_failover() -> Optional[str]:
     'echo "$s" && echo "==Validating launching==" && '
     'echo "$s" | grep -A 1 "Launching on" | grep "is up." && '
     'echo "$s" && echo "==Validating setup output==" && '
-    'echo "$s" | grep -A 1 "Setup detached" | grep "Job submitted" && '
+    'echo "$s" | grep -A 5 "Setup detached" | grep "Job submitted" && '
```

Comment:
Increased to 5 because now there's some additional logs from kubectl port-forward when running on k8s:

<img width="349" height="112" alt="Screenshot 2025-08-16 at 12 18 22 AM" src="https://github.com/user-attachments/assets/9c5aa45e-da8d-410e-b74c-c86031a299dd" />

https://buildkite.com/skypilot-1/smoke-tests/builds/2507/steps/canvas?sid=0198b185-bd24-4684-80fa-72c5f6d7edc1#0198b185-bdf1-4cab-af47-7eef21e8ac1f/56-183